src/main/java/teammates/ui/pagedata/StudentFeedbackResultsPageData.java

```diff
@@ -242,11 +242,12 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
     }
 
     /**
-     * Remove the identifier from anonymous participant name or email, if any.
+     * Remove the anonymous hash from anonymous participant name or email, if any.
      * @param identifier  The given name or email
-     * @return The name or email without any anonymous identifier
+     * @return The name or email without any anonymous hash
      */
-    private String getDisplayableIdentifier(String identifier) {
+    @Deprecated //Slated to be removed
```

Comment:
Add a space after the `//`. Maybe elaborate more: `// The anonymous hash is slated for complete removal`